python-threatexchange/threatexchange/descriptor.py

```diff
@@ -30,6 +28,8 @@ class ThreatDescriptor(t.NamedTuple):
         ...
     }
     """
+    # TODO - do something smarter than this
```

Comment:
More context might be good for when we come back and fix this... e.g. 'Class attribute set by CLI, currently pulled from app_token in main.py (pending refactor).'